### ISSUE TITLE
VOD Snapshot/Code

### ISSUE THREAD

**Issue by WolfgangPonce** [lay295/TwitchDownloader#941]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues?q=is%3Aissue) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Write your feature request here

First of all I wanna thank for the great work you've been doing with the app so far.
I'm a video editor and have a assistant working with my. Basically he trims down the video to a reasonable size and I work from there. So we have to download the same VOD with the same specs for every video.
The solution we came up with is to have the VOD link, the Start and End time on Notion for each video, which is not bad at all, but if there was a code containing all that information it would be better and less likely to messing up the timings.
![Notion 01](https://github.com/lay295/TwitchDownloader/assets/95878427/b9ef85ae-bac5-4ff7-ba27-5a752699f6f2)

So basically my idea is to have a code, kind of like a QRCode works, but this code would contain:
-VOD link
-Quality setting
-Start time
-End time
-OAuth Token
-File name

The idea is that instead of everyone working on the same VOD typing all those settings, as soon as the first person hits "Download" it would generate a code and show a prompt window with it with the buttons "Ok" "Copy Code to Clipboard", add the code to the "Tags" details on the video or even create a text file near the video with the code.
![TAG](https://github.com/lay295/TwitchDownloader/assets/95878427/f36efa84-29dc-49a9-b27b-3649996329f3)

**Comment by ScrubN** [comment]
This is an interesting idea. I do have some concerns though:
- This code is designed to be shared, but OAuth should never be shared. If decide that you trust someone and a messaging service enough to share your OAuth, then that should be your risk to take. I would not appreciate having my account hijacked because I shared a video file that silently contained a code containing my OAuth token.
- The code would probably fit better in the comments rather than the tags, as it is designed to be unique.
- A popup sounds like it could get annoying. This could be mitigated by putting it behind a setting, but I feel a better solution is to just have a dedicated button for it. This would also allow you to creating a settings code without initiating a download.

This is definitely something to consider, but I can't promise a timeline for if/when it will be implemented.
